src/libData/AccountData/AccountStoreSC.tpp

```diff
@@ -729,6 +867,25 @@ bool AccountStoreSC<MAP>::ExportContractFiles(Account& contract) {
     os << DataConversion::CharArrayToString(contract.GetInitData());
     os.close();
 
+    for (const auto& extlib_export : extlibs_exports) {
```

Comment:
This piece of code looks to be the same as the one in `ExportCreateContractFiles`. Would it be a good idea to combine them into a function? (feel free to ignore this suggestion).